tests/acceptance/test_dcos_docker.py

```diff
@@ -0,0 +1,6 @@
+from shakedown import *
+
+def test_docker_version():
+    master_docker_version = docker_version()
+    print("DC/OS master is running Docker Server {}".format(master_docker_version))
```

Comment:
Does all of `shakedown` use `print` statements instead of `logging`? If it uses `logging` elsewhere, I'd say change this, if not leave it. I'll approve the PR regardless.